cpp/lib/include/TimeUtil.h

```diff
@@ -133,6 +133,13 @@ std::string TimeTToString(const time_t &the_time, const std::string &format = DE
                           const std::string &time_locale = UB_DEFAULT_LOCALE);
 
 
+bool StringToTimeT(const std::string &time_str, time_t * const unix_time);
+
+
+/** \note This function will abort if "time_str" cannot be converted to a valid time_t! */
```

Comment:
Please add information about which kinds of strings are allowed.

Reply:
Done!